py/strato/common/log/log2text.py

```diff
@@ -33,17 +37,22 @@
     "\033[2;36m",
 )
 COLOR_OFF = "\033[0;0m"
+LOG_CONFIG_FILE_PATH = "/etc/strato-log.conf"
+HIGHEST_PRIORITY = 0
 
 class Formatter:
     _COLORS = {logging.PROGRESS: CYAN, logging.ERROR: RED, logging.WARNING: YELLOW}
 
     converter = time.gmtime
 
     def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecondPrecision, noColors, localTime=False):
+        self.configFile = yaml.load(open(LOG_CONFIG_FILE_PATH, 'r').read())
         self._firstClock = None
         self._clock = self._relativeClock if relativeTime else self._absoluteClock
         self._relativeClockFormat = "%.6f" if microsecondPrecision else "%.3f"
         self._minimumLevel = logging.INFO if noDebug else logging.DEBUG
+        self._localTimezoneOffset = lineparse.getTimezoneOffset()
+        self._exceptionFiles = {}
```

Comment:
hey, please use meaningful names. 
_exceptionFiles actually is logsFileColorMapping or something similar.